quote_generator.js

```diff
@@ -0,0 +1,15 @@
+const quote_generator = () => {
+    // Storing the API endpoint in a variable
+    const url = "https://type.fit/api/quotes/";
+    // Fetching the API
+    fetch(url)
+        .then((response) => response.json())
+        .then((data) => {
+            // Generate quotes
+            let RandomQuotes = Math.floor(Math.random() * data.length);
+            let quote = data[RandomQuotes].text;
+            console.log(quote);
+        })
+        .catch((err) => console.log(err));
```

Comment:
Error should also be thrown

`throw error`

https://stackoverflow.com/questions/22069532/what-if-i-use-throw-in-catch